app/src/main/res/values/strings.xml

```diff
@@ -771,18 +771,35 @@ Signature: %5$s</string>
     <string name="language">en</string>
     <string name="use_existing">Use existing</string>
 
-
     // search
     <string name="all_results">All results</string>
     <string name="people">People</string>
-    <string name="rooms">Rooms</string>
     <string name="lectures">Lectures</string>
     <string name="search_info">Search for people, rooms, and others.</string>
     <string name="no_result_info">Couldn\'t find “%1$s”</string>
     <string name="no_result_sub_info">Try again, correct spelling or try with different keywords.</string>
     <string name="recent_searches">Recent searches</string>
     <string name="clear_recent_searches">Clear history</string>
     <string name="not_available_search">Not available</string>
+    <string name="n_rooms">Rooms</string>
+    <string name="buildings">Buildings</string>
     <string name="clear_search_history_request">Do you want to clear search history?</string>
 
+    // Navigation details
+    <string name="location_details">Location details</string>
+    <string name="loading_details">loading</string>
+    <string name="site_plans">Site plans</string>
+    <string name="interactive_map">Interactive map</string>
+    <string name="open_location_in_other_app">Open location in other app</string>
+    <string name="available_maps">Available site plans:</string>
+    <string name="map">Map</string>
```

Comment:
map does not seem to be used anywhere else. Is this intentional?

Reply:
My mistake, I should have remove it

Reply:
Ah, it is used in fragment_navigation_details spinner ^^